lib/trello_wrapper.rb

```diff
@@ -82,11 +75,24 @@ def set_description(card_id, description)
 
   private
 
+  def client
```

Comment:
[minor] I think we can safely memoize this method. No need to create a new object every time ```client``` is called.